### DISCUSSION TITLE
Any way to pass options to git log command?

### DISCUSSION THREAD

**Discussion by chiefjester** [PatrickF1/fzf.fish#200] [Q&A] [answered]
Hi, Thank you for this wonderful plugin. 

~~I was wondering how do I search all my git commits, since it just prints just a part of it. I have a very long history. I would want to know how to search till my first commit.~~

~~I know you can pass git log options to fzf, but I don't see any option to pass so I can have the full git log history.~~

I wanted to delete this  (seeing the whole history now), but couldn't so I just leave my previous question. I guess I want to ask is what options can I pass on `fzf` to change the git log function?

**Comment by PatrickF1** [comment, marked as answer]
Hi Chris, that's odd and unexpected behavior. When you git Ctrl+Option+L, that should be searching your entire git history. How do you know it's only searching part of it?

**Reply by chiefjester** [reply]
I had some missing logs, but I forgot I was in a new branch. Been rewriting a repo before I publish it as public (removing some sensitive data).

**Reply by PatrickF1** [reply]
Great to hear! Would you mind starring this repo?

> what options can I pass on fzf to change the git log function?

There isn't a feature for that right now. You can only pass options to fzf for the git log feature https://github.com/PatrickF1/fzf.fish#pass-fzf-options-for-a-specific-feature

**Reply by chiefjester** [reply]
of course, sorry forgot about it. Starred now.
